zerver/webhooks/wekan/tests.py

```diff
@@ -0,0 +1,124 @@
+                                            content_type="application/x-www-form-urlencoded")
+
+    def test_delete_attachment_message(self) -> None:
+        expected_message = u"eyenn1 delete attachment at board \"Bucket List\".\n\n[See in Wekan](http://127.0.0.1/b/Jinj4Xj7qnHLRmrTY/bucket-list/pMtu7kPZvMuhhC4hL)"
+        self.send_and_test_stream_message('delete_attachment', u'Wekan Notification' , expected_message,
+                                            content_type="application/x-www-form-urlencoded")
+
+    def test_join_member_message(self) -> None:
+        expected_message = u"eyenn1 join member at board \"Bucket List\".\n\n[See in Wekan](http://127.0.0.1/b/Jinj4Xj7qnHLRmrTY/bucket-list/TMmjFnQGuZPsbjXzS)"
+        self.send_and_test_stream_message('join_member', u'Wekan Notification' , expected_message,
+                                            content_type="application/x-www-form-urlencoded")
+
+    def test_move_card_message(self) -> None:
+        expected_message = u"eyenn1 move card at board \"Bucket List\".\n\n[See in Wekan](http://127.0.0.1/b/Jinj4Xj7qnHLRmrTY/bucket-list/pMtu7kPZvMuhhC4hL)"
+        self.send_and_test_stream_message('move_card', u'Wekan Notification' , expected_message,
+                                            content_type="application/x-www-form-urlencoded")
+
+    def test_remove_list_message(self) -> None:
+        expected_message =  u"eyenn1 remove list at board \"Bucket List\".\n\n[See in Wekan](http://127.0.0.1/b/Jinj4Xj7qnHLRmrTY/bucket-list)"
+        self.send_and_test_stream_message('remove_list', u'Wekan Notification' , expected_message,
+                                            content_type="application/x-www-form-urlencoded")
+
+    def test_remove_swimlane_message(self) -> None:
+        expected_message = u"eyenn1 remove swimlane at board \"Bucket List\".\n\n[See in Wekan](http://127.0.0.1/b/Jinj4Xj7qnHLRmrTY/bucket-list)"
+        self.send_and_test_stream_message('remove_swimlane', u'Wekan Notification' , expected_message,
+                                            content_type="application/x-www-form-urlencoded")
+
+    def test_removed_checklist_item_message(self) -> None:
+        expected_message = u"eyenn1 removed checklist item at board \"Bucket List\".\n\n[See in Wekan](http://127.0.0.1/b/Jinj4Xj7qnHLRmrTY/bucket-list/pMtu7kPZvMuhhC4hL)"
+        self.send_and_test_stream_message('removed_checklist_item', u'Wekan Notification' , expected_message,
+                                            content_type="application/x-www-form-urlencoded")
+
+    def test_removed_checklist_message(self) -> None:
+        expected_message = u"eyenn1 remove checklist at board \"Bucket List\".\n\n[See in Wekan](http://127.0.0.1/b/Jinj4Xj7qnHLRmrTY/bucket-list/pMtu7kPZvMuhhC4hL)"
+        self.send_and_test_stream_message('removed_checklist', u'Wekan Notification' , expected_message,
+                                            content_type="application/x-www-form-urlencoded")
```

Comment:
Generally, we write our strings to read as natural English; so this should be `eyenn1 removed a checklist...`

Also, is there a label or something we can link to in order to indicate the specific "checklist" or "swimlane" that was modified when one is added?  

In any case, I think rather than doing '...at board "Bucket List"...' we should refer to it as '... in board [Bucket List](http...)', taking advantage of markdown links.

Reply:
I have taken note of these points. I am working on them now. I will update when I am done or need more clarification.